<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YJChatServerClass</class>
 <widget class="QMainWindow" name="YJChatServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YJChatServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
      <item>
       <widget class="QTextBrowser" name="m_textBrowser"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,1">
          <item>
           <widget class="QLabel" name="m_serverIPLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Server IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_IP_lineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="m_serverPortLabel">
            <property name="text">
             <string>Server Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_port_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="m_tableWidget">
          <column>
           <property name="text">
            <string>Username</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Hostname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP Adress</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="yjchatserver.qrc"/>
 </resources>
 <connections/>
</ui>
